spec/chore/queues/spec_helper.rb

```diff
@@ -0,0 +1,20 @@
+require 'spec_helper'
+
+module Chore
+  RSpec.shared_context 'fake objects' do
+    let(:job) { {'class' => 'TestJob', 'args'=>[1,2,'3']}}
```

Comment:
This seems like an outlier that doesn't need to be in the shared context.  I can see an argument for the more complex s3 class creation being shared because you're effectively creating a fake_sqs, though.